src/dashboard/domain/scores/readability-scores/readability-scores-interface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+// phpcs:disable Yoast.NamingConventions.NamespaceName
+namespace Yoast\WP\SEO\Dashboard\Domain\Scores\Readability_Scores;
+
+use Yoast\WP\SEO\Dashboard\Domain\Scores\Scores_Interface;
+
+/**
+ * This interface describes a readability score implementation.
+ */
+interface Readability_Scores_Interface extends Scores_Interface {}
```

Comment:
I did it mainly so that I can gather all `Readability_Scores_Interface` implementations and only those, in `Readability_Scores_Repository`. But it does add clarity as well.